src/Servers/Kestrel/Core/src/KestrelServer.cs

```diff
@@ -93,7 +86,7 @@ public void Initialize(IHostEnvironment hostEnvironment, ILogger<KestrelServer>
 
         public void PopulateMultiplexedTransportFeatures(FeatureCollection features, ListenOptions listenOptions)
         {
-            HttpsConfigurationService.PopulateMultiplexedTransportFeaturesWorker(features, listenOptions, _httpsLogger);
```

Comment:
This is where I thought it might be used: https://github.com/dotnet/aspnetcore/blob/c2d95a80e60c20dd6ec9a7debc522c47782b2bf5/src/Servers/Kestrel/Core/src/Internal/Infrastructure/TransportManager.cs#L75

Reply:
If we need to keep it, Brennan suggested the possibility of passing the type to the logger by name, rather than as a type argument.

Reply:
Fortunately, we throw right before reaching that line if there is no `IMultiplexedConnectionListenerFactory` available and there never should any be when using the `KestrelServer` constructor.

https://github.com/dotnet/aspnetcore/blob/c2d95a80e60c20dd6ec9a7debc522c47782b2bf5/src/Servers/Kestrel/Core/src/Internal/Infrastructure/TransportManager.cs#L67-L75

https://github.com/dotnet/aspnetcore/blob/c2d95a80e60c20dd6ec9a7debc522c47782b2bf5/src/Servers/Kestrel/Core/src/KestrelServer.cs#L31-L39

Reply:
That's what I was missing.  Thanks!